### DISCUSSION TITLE
Dialogic.Save.load() automatically starts a new timeline, and block mouse UI Inputs

### DISCUSSION THREAD

**Discussion by SXijka** [dialogic-godot/dialogic#2059] [Q&A]
Dialogic.Save.load() will automatically start a new timeline, which will create an instance of default_layout_base.tscn and add it to the scene tree. I only wanted to load some stored variables, but a CanvasLayer node named DialogicLayout_VisualNovelStyle added to the tree blocked my mouse UI Inputs.
 Dialogic.end_timeline() can remove the node, so the problem fixed, but I still wonder:
With Dialogic.start(), I have to give it a parameter which is the name of the timeline started. But with Dialogic.Save.load(), it doesn't require a parameter and always starts a timeline even when the saved didn't contain any timeline. And I wonder what the timeline started by the load() is?

**Comment by Jowan-Spooner** [comment]
Hmm... It should only start a timeline if a timeline was playing when Dialogic.Save.save() was called. 
It might be a bug if it does indeed start a timeline even if no timeline had been playing when you saved. 
Could you share the save file that results in this problem?

**Reply by SXijka** [reply]
[state.txt](https://github.com/dialogic-godot/dialogic/files/14142363/state.txt)

**Reply by SXijka** [reply]
I found out that the current timeline is null, but load() does add a DialogicLayout_VisualNovelStyle node to the tree and block mouse Input, and somehow end_timeline() can remove this node. Is this function originally designed to work this way?
